src/components/screens/ServiceTile.tsx

```diff
@@ -58,15 +61,22 @@ export const ServiceTile = ({
     [isEditMode, isVisible]
   );
   const ToggleVisibilityIcon = isVisible ? Icon.Visible : Icon.Unvisible;
+  const { fontStyle = {}, iconStyle = {}, numberOfLines, tileStyle = {} } = serviceTileStyles;
+
+  const hasFontStyle = !!Object.keys(fontStyle).length;
+  const hasIconStyle = !!Object.keys(iconStyle).length;
+  const hasTileStyle = !!Object.keys(tileStyle).length;
 
   return (
     <ServiceBox
-      orientation={orientation}
+      bigTile={!!item.tile || hasTileStyle}
```

Comment:
this reads weird, if there is a tile or it has tile style then it is a big tile? having tile style always means big tile?
what about having a variable `isBigStyle` with some comments, to clarify the logic?

Reply:
Yes. If `tileStyle` is available, it always means bigTile. Until now we had no other option to create a bigTile than adding an image. We are trying to add the option to create the bigTile from the main-server. If there is no `tileStyle` then this is a normal service tile.

Reply:
ok thanks for clearing out.
and what is with the second part?
> what about having a variable isBigStyle with some comments, to clarify the logic?